docs/UserGuide.md

```diff
@@ -404,8 +401,41 @@ Furthermore, certain edits can cause the TeachStack to behave in unexpected ways
 
 ### 7.14 Setting weak threshold
```

Comment:
Do you think adding the command at the end would make this more consistent with the other features, ie.

### 7.14 Setting weak threshold : `setweak`

Reply:
Done in latest pr